packages/ui-react/src/IdentityIcon/Substrate.tsx

```diff
@@ -12,7 +12,7 @@ import isU8a from '@polkadot/util/is/u8a';
 
 type Props = BaseProps & {
   size: number,
-  value: string | Uint8Array
+  value: string | null
```

Comment:
As above, let's allow at the base-level for both. (Especially around short-addresses, not 100% sure how they would look, plus if we share these libraries, we want to allow some flexibility)